lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -146,6 +146,14 @@
                your playbook accordingly to not export the disk all the time.
                This option is valid only for template disks."
         version_added: "2.4"
+    host:
```

Comment:
Could a more descriptive name be given ? I am not sure people would assume that giving the host will trigger a refresh (at least I don't, but I do not know ovirt well enough).

Reply:
Is `host` the term that ovirt uses?
Just want to ensure their isn't confusion between what Ansible refers to as a host vs ovirt

Reply:
Yeah, but so, could it be renamed, or does it make sense for ovirt users ? Again, i am sysadmin and I find that naming to be confusing (as I guess "host" is used for the host to connect usually) , but I am not using ovirt, so maybe it is obvious for people who are using it.

Reply:
It's correct for oVirt/RHV, by host we mean a machine where VMs are running.